### DISCUSSION TITLE
does fluss have concept of topic, partitions, brokers, controllers etc like kafka does?

### DISCUSSION THREAD

**Discussion by ethany21** [apache/fluss#104] [Q&A] [answered]
If fluss is similar to kafka (with columnar format), then does fluss have notion of partions, topic, brokers etc 
as kafka does?

**Comment by polyzos** [comment, marked as answer]
In Fluss you have tables instead of topics:
- Log Tables is similar to Kafka append only log and you can also guarantee isolation on the bucket level which is similar to a topic's partition.
- In Fluss you also have Primary Key Tables, to handle updates and also perform fast lookup queries on data streams

Instead of brokers, you have the TabletServer which is similar, see [here](https://alibaba.github.io/fluss-docs/docs/concepts/architecture/)
